sdk/storage/Azure.Storage.Files/samples/Sample01a_HelloWorld.cs

```diff
@@ -31,9 +31,7 @@ public void Upload()
             // obtain your connection string from the Azure Portal (click
             // Access Keys under Settings in the Portal Storage account blade)
             // or using the Azure CLI with:
-            // 
             //     az storage account show-connection-string --name <account_name> --resource-group <resource_group>
```

Comment:
As we chatted about - I like these lines to help make the shell command clearer.